internal/engine/local.go

```diff
@@ -823,18 +823,27 @@ func (g *GoliacLocalImpl) SyncUsersAndTeams(ctx context.Context, repoconfig *con
 		return false
 	}
 
+	//
+	// let's update repositories
+	//
+	reposchanged, err := entity.ReadAndAdjustRepositories(w.Filesystem, "archived", "teams", g.users, g.externalUsers)
+	if err != nil {
+		logsCollector.AddError(err)
+		return false
+	}
+
 	// check if we have too many changesets
-	if !force && len(teamschanged)+len(deletedusers)+len(addedusers) > repoconfig.MaxChangesets {
-		logsCollector.AddError(fmt.Errorf("too many changesets (%d) to commit. Please increase max_changesets in goliac.yaml", len(teamschanged)+len(deletedusers)+len(addedusers)))
+	if !force && len(teamschanged)+len(reposchanged)+len(deletedusers)+len(addedusers) > repoconfig.MaxChangesets {
+		logsCollector.AddError(fmt.Errorf("too many changesets (%d) to commit. Please increase max_changesets in goliac.yaml", len(teamschanged)+len(reposchanged)+len(deletedusers)+len(addedusers)))
 		return false
 	}
 
 	//
 	// let's commit
 	//
-	if len(teamschanged) > 0 || len(deletedusers) > 0 || len(addedusers) > 0 {
+	if len(teamschanged) > 0 || len(reposchanged) > 0 || len(deletedusers) > 0 || len(addedusers) > 0 {
```

Comment:
nitpick, but maybe could be bump up as a single sum of changes to be used in both `if` line 836 and 846